recirq/quantum_chess/experiments/batch_moves.py

```diff
@@ -52,7 +52,17 @@ def apply_moves(b: ab.AsciiBoard, moves: List[str]) -> bool:
 def main_loop(args):
     f = open(args.filename, 'r')
     moves = [line.strip() for line in f]
-    board = create_board(processor_name=args.processor_name,
+    if args.processor_name:
+        processor_name = args.processor_name
+    else:
+        # Execute on a quantum processor if it is available.
```

Comment:
Let's update the docstring above to explain the new behavior.

Reply:
done